# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../djbdns/x-cross.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


We can not run those executables on cross builds ..

  - Rene Rebe <rene@exactcode.de>

--- djbdns-1.05/Makefile	2006-01-03 15:26:01.000000000 +0100
+++ djbdns-1.05/Makefile	2006-01-03 15:28:33.000000000 +0100
@@ -31,7 +31,7 @@
 
 auto_home.c: \
 auto-str conf-home
-	./auto-str auto_home `head -n 1 conf-home` > auto_home.c
+	#./auto-str auto_home `head -n 1 conf-home` > auto_home.c
 
 auto_home.o: \
 compile auto_home.c
@@ -511,8 +511,8 @@
 hasshsgr.h: \
 choose compile load tryshsgr.c hasshsgr.h1 hasshsgr.h2 chkshsgr \
 warn-shsgr
-	./chkshsgr || ( cat warn-shsgr; exit 1 )
-	./choose clr tryshsgr hasshsgr.h1 hasshsgr.h2 > hasshsgr.h
+	#./chkshsgr || ( cat warn-shsgr; exit 1 )
+	cat hasshsgr.h2 > hasshsgr.h
 
 hier.o: \
 compile hier.c auto_home.h
